axelrod/strategies/_filters.py

```diff
@@ -0,0 +1,179 @@
+from distutils.util import strtobool
+from collections import namedtuple
+import operator
+
+
+def passes_boolean_filter(strategy, classifier_key, value):
+    """
+    Tests whether a given strategy passes a boolean equality filter for a
+    given key in its classifier dict.
+
+    Parameters
+    ----------
+        strategy : a descendant class of axelrod.Player
+        classifier_key: string
+            Defining which entry from the strategy's classifier dict is to be
+            tested (e.g. 'stochastic' or 'inspects_source').
+        value: string or boolean
```

Comment:
Could that be dealt with in the api? So the api, would convert the string to a boolean? Basically just this `filter_value = strtobool(value)` would need to be in the api right?

Just thinking that it feel a bit strange in the library, we could also pass integers that would be converted to booleans right? (Not that we'd want to). From the library's point of view we're only expecting booleans here.